coriolis/osmorphing/osmount/windows.py

```diff
@@ -105,6 +110,30 @@ def _service_disks_with_status(
                             raise
                     break
 
+    def _get_disk_ids_from_drive_letters(self, drive_letters):
+        disk_ids = []
+        volume_entry_re = r"\s+Volume ([0-9]+)\s+([A-Z]+)\s+"
+        volume_list_script = "LIST VOLUME\r\nEXIT"
+        vol_list = self._run_diskpart_script(volume_list_script)
```

Comment:
Small nit but please move these two declarations right before the `for volume in vol_list...` since that's where they're used.